_pages/en/strategy-version-control-systems.md

```diff
@@ -0,0 +1,346 @@
+sections: Strategies
+version: 0.01
+date: "2020-11-05"
+permalink: /strategy-version-control-systems.html
+---
+
+<!-- markdownlint-disable MD033 -->
+<!-- the below cSpell statement says to ignore any text between HTML tags. E.g. it will ignore "th rowspan='2'" in this string: <th rowspan='2'> -->
+<!-- cSpell:ignoreRegExp /\<[^\>]+\>/ -->
+
+<style>
+table, th, td {
+  border: 1px solid black;
+}
+
+th {
+  background-color: #ccccff;
+}
+</style>
+
+<div class="alert alert-warning">
+  <strong>This is a DRAFT strategy</strong>.
+</div>
+
+<!-- markdownlint-disable MD001 -->
+##### Table of Content <!-- omit in toc -->
+<!-- markdownlint-enable MD001 -->
+- [Introduction](#introduction)
+  - [Purpose](#purpose)
+  - [Target Audience](#target-audience)
+  - [Business Case](#business-case)
+- [Guiding Policy](#guiding-policy)
+- [Coherent set of actions](#coherent-set-of-actions)
+- [Measuring the Strategy's success](#measuring-the-strategys-success)
+- [Appendix A - Definitions](#appendix-a---definitions)
+- [Appendix B - Acronym List Definition](#appendix-b---acronym-list-definition)
+
+## Introduction
+
+### Purpose
+
+To provide IITB with modern and supported **Version Control Systems (VCS)** for use in software projects, and a roadmap to get there.
+
+This strategy is aimed at reducing the need to support various legacy version control systems, while increasing the collaboration and visibility between projects within ESDC, across the GC and publicly.
+
+The strategy includes:
+
+1. A **Guiding Policy**, which serves to set automatic decisions that defines the expected usage of the version control systems
+2. A **Coherent set of actions** (an action plan), which serve to operationalize the version control systems
+
+The intent behind this strategy is to communicate a decision by the CIO (not yet approved) on a path forward (the Guiding Policy), and what investments are needed to operationalize that decision (the coherent set of actions).
+
+### Target Audience
+
+This **strategy document** is targeted to stakeholders involved in determining how IT Products are delivered.
+More specifically, stakeholders involved in defining the rules for building, delivering, operationalizing, and maintaining IT Products.
+The stakeholders are listed in section [Coherent set of actions](#coherent-set-of-actions) and are expected to participate in the execution stage of this strategy necessary to operationalize the Guiding Policy.
+
+The **Guiding Policy**, once operationalized, will define the default systems where ESDC IITB will publish and maintain its source code, scripts, and system/database configurations.
+All ESDC personnel involved in the management of IT Products is expected to adhere to this policy, including but not limited to, development and operations staff.
+
+### Business Case
+
+Moving to the digital age requires improving IT's responsiveness.
+
+To improve IT's responsiveness, we must find ways to reduce risks associated with its use.
```

Comment:
Do we go into what risks come from using multiple VCS, including no VCS at all? Or do we keep that for a fully fleshed out business case?

Reply:
The Benefits section kind of show the "opposite" of the risks ;)
I would keep the details for a fully fleshed out business case.